.eslintrc.js

```diff
@@ -26,5 +28,37 @@ module.exports = {
         'node/no-unpublished-require': 'off',
       },
     },
+    {
+      files: ['**/*.ts'],
+      extends: [
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended',
+        'prettier/@typescript-eslint',
```

Comment:
I don't think you need this any more (with latest release of eslint-config-prettier)